app/containers/App/index.js

```diff
@@ -26,22 +39,37 @@ const AppWrapper = styled.div`
   flex-direction: column;
 `;
 
-export default function App() {
-  return (
-    <AppWrapper>
-      <Helmet
-        titleTemplate="%s - React.js Boilerplate"
-        defaultTitle="React.js Boilerplate"
-      >
-        <meta name="description" content="A React.js Boilerplate application" />
-      </Helmet>
+export const App = ({ handleError: errorHandler }) => (
+  <AppWrapper>
+    <Helmet
+      titleTemplate="%s - React.js Boilerplate"
+      defaultTitle="React.js Boilerplate"
+    >
+      <meta name="description" content="A React.js Boilerplate application" />
+    </Helmet>
+    <ErrorBoundary onError={errorHandler} FallbackComponent={<ErrorFallback />}>
```

Comment:
Should we have `ErrorBoundary` here or above the `AppWrapper`? I think any errors in components higher in the tree will not be caught by this boundary.

Reply:
Yeah, probably higher up.